i18n/fr/docusaurus-plugin-content-docs/version-23.04/upgrade/upgrade-from-20-10.md

```diff
@@ -412,8 +463,7 @@ accéder à la page de connexion :
 
 > La présentation de l'interface ayant été modifiée dans la version 22.10, vous devez vider le cache de votre navigateur pour afficher le nouveau thème.
 
-Si le module Centreon BAM est installé, référez-vous à la [documentation
-associée](../service-mapping/upgrade.md) pour le mettre à jour.
+Référez-vous à la documentation de mise à jour pour [Centreon BAM](../service-mapping/upgrade.md), [Centreon MBI](../reporting/update.md) et [Centreon MAP](../graph-views/map-web-upgrade.md) pour mettre à jour ces modules.
```

Comment:
BAM is supposed to be upgraded before connecting to the interface (the correct link is to the upgrade topic, not the update one)
![image](https://github.com/user-attachments/assets/614b48ba-95bf-482a-ad62-d56dbb664086)